priv/repo/migrations/.formatter.exs

```diff
@@ -1,4 +1,4 @@
 [
   import_deps: [:ecto_sql],
-  inputs: ["*.exs"]
+  inputs: ["*.exs", "*.heex"]
```

Comment:
it's useless add this. As far as I know mix format does not change any .heex template